go.mod

```diff
@@ -156,11 +156,16 @@ replace (
 	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.4
 
 	// TODO: remove when middleware will be implemented
-	github.com/cosmos/ibc-go/v2 => github.com/crypto-org-chain/ibc-go/v2 v2.2.0-hooks2
+	github.com/cosmos/ibc-go/v3 => github.com/crypto-org-chain/ibc-go/v3 v3.0.0-hooks
 
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
-	github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.10.0-alpha1-cronos-9
+	// TODO: remove when gravity update dependencies
+	github.com/peggyjv/gravity-bridge/module/v2 => github.com/crypto-org-chain/gravity-bridge/module/v2 v2.0.0-20220421052611-ae947806716e
```

Comment:
can you update to use this branch instead?
https://github.com/crypto-org-chain/gravity-bridge. 

(v2.0.0-cronos, commit a016e2b04866bab93c106abb0b4479bfc788f03f )

This is going to be used for the audit